tests/_support/Helper/Guzzle.php

```diff
@@ -147,4 +147,298 @@ public function _after(TestInterface $test)
             @unlink($file);
         }
     }
+
+    public function pdf2text($filename)
```

Comment:
Probably better to move this (and the later methods) to a new helper for PDF parsing as it isn't related to guzzle.